### PR TITLE
Master

### PR DESCRIPTION
**Pull request by knperi**
Adding new  Mezmo (LogDNA) Public, Private and Ingestion endpoints

### CODE REVIEW

**Comment by knperi** [comment]
@bradbehle @derekpoindexter @kersten1 

Please can you review the changes I have made. The upcoming changes are documented [here](https://test.cloud.ibm.com/docs/log-analysis?topic=log-analysis-endpoints)

**Review by bradbehle — CHANGES_REQUESTED** [review]
What you have here looks right, just a few things I noticed:

1. Please remove those binary .DS_Store files from the commit/PR
2. These IPs I think should also be in the allow-private-services.yaml and allow-public-services.yaml files (the non "pod" policies) since the old/existing LogDNA IPs are in both
3. Do you know if IKS/ROKS clusters with workers in Chennai use the jp-tok or Chennai addresses?  If they do use the Chennai specific ones, then the new Chennai specific IPs should go in the jp-tok policies (along with the jp-tok specific IPs)

**Comment by knperi** [comment]
@bradbehle Removed .DS_Store files and modified allow-<privare|public>-services.yaml files as suggested. Please review.

File: calico-policies/public-network-isolation/jp-tok/allow-public-services-pods.yaml [hunk 1]
```diff
@@ -77,10 +77,15 @@ spec:
...
+      # New IP addresses added by perik@us.ibm.com 1/19/2023
+      - 161.202.231.186/32
+      - 128.168.96.179/32
+      - 165.192.111.36/32
+      # End new IP address list
       ports:
       - 443
       - 80
-    protocol: TCP
+    prEtocol: TCP
```
Comment (bradbehle) [inline calico-policies/public-network-isolation/jp-tok/allow-public-services-pods.yaml:88]: typo I assume?

**Review by bradbehle — CHANGES_REQUESTED** [review]
Looks like one typo, and still two .DS_Store binaries to remove

**Comment by knperi** [comment]
@bradbehle Sorry for the typo. Fixed and removed the other binary files

File: calico-policies/public-network-isolation/eu-de/allow-public-services-pods.yaml [hunk 2]
```diff
@@ -63,6 +63,11 @@ spec:
       - 158.177.129.36/32
       - 161.156.89.11/32
       - 161.156.89.12/32
+      # New IP addresses added by perik@us.ibm.com 1/19/2023
```
Comment (bradbehle) [inline calico-policies/public-network-isolation/eu-de/allow-public-services-pods.yaml:66]: In the docs here: https://cloud.ibm.com/docs/log-analysis?topic=log-analysis-endpoints  these IPs are listed as for ca-tor, so I think there might have been a copy/paste error here?

File: calico-policies/public-network-isolation/eu-de/allow-public-services.yaml [hunk 3]
```diff
@@ -63,6 +63,11 @@ spec:
       - 158.177.129.36/32
       - 161.156.89.11/32
       - 161.156.89.12/32
+      # New IP addresses added by perik@us.ibm.com 1/19/2023
```
Comment (bradbehle) [inline calico-policies/public-network-isolation/eu-de/allow-public-services.yaml:66]: In the docs here: https://cloud.ibm.com/docs/log-analysis?topic=log-analysis-endpoints  these IPs are listed as for ca-tor, so I think there might have been a copy/paste error here?

**Review by bradbehle — CHANGES_REQUESTED** [review]

**Comment by knperi** [comment]
@bradbehle Yes. It was a cut and paste error.

**Review by bradbehle — APPROVED** [review]

**Review by mihivagyok — COMMENTED** [review]
Please use better PR name in the future. Also, are you sure you want your email address in the files? By this, anyone can read it easily, and the address could be added to be a spam database. We could use git history instead. Thanks!

**Review by derekpoindexter — APPROVED** [review]

**Pull request closed without merging**